src/ffi/cpython/pystate.rs

```diff
@@ -1,7 +1,17 @@
-use crate::ffi::pystate::{PyInterpreterState, PyThreadState};
+use crate::ffi::{PyFrameObject, PyInterpreterState, PyObject, PyThreadState};
 use std::os::raw::c_int;
 
-// Py_tracefunc is defined in ffi::pystate
+// skipped _PyInterpreterState_RequiresIDRef
+// skipped _PyInterpreterState_RequireIDRef
+
+// skipped _PyInterpreterState_GetMainModule
+
+pub type Py_tracefunc = extern "C" fn(
```

Comment:
These functions were in `Include/pystate` before 3.8. The problem with moving this here now is that the `cpython/pystate` module is gated behind `Py_3_8`, so this removes it from 3.6 and 3.7, which is why CI is now failing. (the same thing may be true for other things)

Reply:
Ah. I'd rather have the ffi modules look as much like the Python headers as possible, to minimise confusion. So I'll remove that version gating and tweak things.